JavadocStyleGuide.md

```diff
@@ -0,0 +1,242 @@
+# Gradle Javadoc Style Guide
+
+## 1.1 Formatting
+
+### 1.1.1 General form
+
+The basic formatting of Javadoc blocks is as seen in this example:
+
+```java
+/**
+ * Returns an Image object that can then be painted on the screen.
+ * <p>
+ * The url argument must specify an absolute {@link URL}. 
+ * The name argument is a specifier that is relative to the url argument.
+ * This method always returns immediately, whether or not the image exists. 
+ *
+ * @param url an absolute URL giving the base location of the image
+ * @param name the location of the image, relative to the url argument
+ * @return the image at the specified URL
+ * @see Image
+ */
+public Image getImage(URL url, String name) {
+   try {
+       return getImage(new URL(url, name));
+   } catch (MalformedURLException e) {
+       return null;
+   }
+}
+```
+
+Basic formatting rules:
+
+- The first line contains the begin-comment delimiter ( `/**`).
+- The first sentence is a summary.
+- Notice the inline tag `{@link URL}`, which converts to an HTML hyperlink pointing to the documentation for the URL class.
+- If you have more than one paragraph in the doc comment, separate the paragraphs with a `<p>` paragraph tag, as shown.
+- Insert a blank comment line between the description and the list of tags, as shown.
+- The first line that begins with an `@` character ends the description; you cannot continue the description following block tags.
+- Block tags must be added in order.
+- The last line contains the end-comment delimiter ( `*/`).
+
+So lines won't wrap, limit any doc-comment lines to 80 characters.
```

Comment:
Can we increase this to 100 or 120?  80 is so low and monitors are bigger these days.